code/framework/src/scripting/builtins/quaternion.h

```diff
@@ -116,8 +138,10 @@ namespace Framework::Scripting::Builtins {
             cls["y"] = sol::property([](const Quaternion& self) { return self.GetY(); });
             cls["z"] = sol::property([](const Quaternion& self) { return self.GetZ(); });
             cls["length"] = sol::property([](const Quaternion& self) { return self.GetLength(); });
-            cls["toString"] = &Quaternion::ToString;
+            cls["__tostring"] = &Quaternion::ToString;
```

Comment:
Please do not change this signature.

Reply:
I changed it because it make possible to do just `print(quaternion)` instead `print(quaternion:toString())`